packages/TelegramClient-Core.package/TCCBasicGroupChat.class/instance/loadInfo.st

```diff
@@ -1,11 +1,11 @@
 loading
 loadInfo
 
-	| callbackId |
-	callbackId := self core registerCallback: [:aBasicGroupFullInfoEvent |
-		self
-			memberCount: (aBasicGroupFullInfoEvent at: 'members') size;
-			description: (aBasicGroupFullInfoEvent at: 'description');
-			addMembersFrom: (aBasicGroupFullInfoEvent at: 'members').
-		].
-	self core send: (TCCRequest newGetBasicGroupFullInfoOf: self groupId with: callbackId).
\ No newline at end of file
+	self core
+		send: (TCCRequest newGetBasicGroupFullInfoOf: self groupId with: 'this should work')
+		thenDo: [:aBasicGroupFullInfoEvent |
+			self
```

Comment:
Given that this is not a test, why is the text `'this should work'` there? :D

Reply:
Leftovers from testing :D